randovania/server/discord/preset_lookup.py

```diff
@@ -173,15 +173,14 @@ async def reply_for_preset(message: discord.Message, versioned_preset: Versioned
 
 async def reply_for_layout_description(message: discord.Message, description: LayoutDescription):
     embed = discord.Embed(
-        title=f"Spoiler file for Randovania {description.randovania_version_text}",
+        title=f"Spoiler file (Generated in {description.randovania_version_text})",
```

Comment:
I know I suggested this wording, but I'm wondering if "in" is correct. Would "with" be better?